sklearn/tests/test_calibration.py

```diff
@@ -401,6 +426,90 @@ def test_sigmoid_calibration():
         _SigmoidCalibration().fit(np.vstack((exF, exF)), exY)
 
 
+@pytest.mark.parametrize(
+    "clf",
+    [LogisticRegression(penalty=None, tol=1e-6, random_state=0)],
+)
```

Comment:
As this is only one estimator, we can remove the parametrize and put it inside the test function.